src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -350,7 +350,7 @@
 
         <service id="sulu_security.user_manager" class="Sulu\Bundle\SecurityBundle\UserManager\UserManager" public="true">
             <argument type="service" id="doctrine.orm.entity_manager"/>
-            <argument type="service" id="security.encoder_factory" on-invalid="null"/>
+            <argument type="service" id="security.password_hasher_factory" on-invalid="null"/>
```

Comment:
We are currently still supporting Symfony 4.4 and so we can not yet use the new password hasher service as that does only exist since 5.3.

Reply:
Ou than at all until 6.0 forgot about it :))) people not update their sf projects 🤔

Reply:
Still thank you for your afford!